db/migrate/20240227071401_add_total_calories_to_dishes.rb

```diff
@@ -0,0 +1,5 @@
+class AddTotalCaloriesToDishes < ActiveRecord::Migration[7.1]
+  def change
+    add_column :dishes, :total_calories, :integer
```

Comment:
Instead of adding this column to the dish table, we were instead looking for you to do a sum of all the calories of a dish's ingredients. If `self` is a dish instance in a model instance method, `self.ingredients.sum(:calories)` would give us the total of all the calories for a given dish